eng/build.ps1

```diff
@@ -61,6 +61,7 @@ if ($help -or (($null -ne $properties) -and ($properties.Contains('/help') -or $
 
 # VS Test Explorer support for libraries
 if ($vs) {
+  Write-Host "!!! VS Test Explorer now works without the -vs switch. The switch will be removed eventually. !!! "
```

Comment:
maybe mention it will be removed on `June`? instead of `Eventually`?

Reply:
I will definitely not write a date down here :D Consider we discover a blocker and can't remove the switch. Eventually is the best that I can do 😂